src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Orchestrators/EmployerRequestOrchestrator.cs

```diff
@@ -412,22 +407,16 @@ public void UpdateTrainingOptionsForEmployerRequest(EnterTrainingOptionsEmployer
 
         public async Task<CheckYourAnswersEmployerRequestViewModel> GetCheckYourAnswersEmployerRequestViewModel(SubmitEmployerRequestParameters parameters, ModelStateDictionary modelState)
         {
-            var standard = await _mediator.Send(new GetStandardQuery(parameters.StandardId));
-            if (standard == null)
-            {
-                throw new ArgumentException($"The standard {parameters.StandardId} was not found");
-            }
-
             var employerRequest = SessionEmployerRequest;
 
             return new CheckYourAnswersEmployerRequestViewModel
             {
                 HashedAccountId = parameters.HashedAccountId,
-                StandardId = parameters.StandardId,
-                RequestType = parameters.RequestType,
-                Location = parameters.Location,
-                StandardTitle = standard.Title,
-                StandardLevel = standard.Level,
+                StandardReference = SessionEmployerRequest.StandardReference,
```

Comment:
The base view model of CheckYourAnswersEmployerRequestViewModel is SubmitEmployerRequestViewModel - which still has a StandardId - which is being used but now not being set

I think SubmitEmployerRequestViewModel needs a StandardReference and a StandardLarsCode not a StandardId - then it should make more sense what is what

Reply:
replaced StandardId with StandardReference and LarsCode